storage/service_account/hmac/hmac_test.go

```diff
@@ -79,7 +79,7 @@ func TestCreateKey(t *testing.T) {
 
 func TestActivateKey(t *testing.T) {
 	tc := testutil.SystemTest(t)
-	key, err := createTestKey(tc.ProjectID)
+	key, err := createTestKey(tc.ProjectID, t)
 	defer deleteTestKey(key)
 	if err != nil {
 		t.Errorf("Error in key creation: %s", err)
```

Comment:
This (and all similar checks for `createTestKey`) should probably be `t.Fatalf`. If key creation failed, I don't think it makes sense to continue?

(`t.Fatalf` ends the test immediately. `t.Errorf` continues execution.)

Reply:
I thought about that (failing immediately in the createTestKey function)..  @tritone mentioned we could probably just leave it as is since the key creation step is just a setup for the sample tests